### ISSUE TITLE
edema localized to joint (DIP, PIP, MCP)

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10705]
**Preferred term label:**
edema localized to joint (DIP, PIP, MCP)

**Synonyms**
Distal interphalangeal (DIP) joint, proximal interphalangeal (PIP) joint, metacarpophalangeal (MCP) joint

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Interphalangeal joint sprain
Proximal interphalangeal (PIP) joint dislocation
Distal interphalangeal (DIP) joint dislocation
Metacarpophalangeal (MCP) joint arthritis

**Your nano-attribution (ORCID)**
